nextjs-task-system/app/api/tasks/route.tsx

```
@@ -0,0 +1,132 @@
+    const filters: FormatedFilters = {
+      typeOfAssigned: searchParams.get("typeOfAssigned") || "",
+      assignedUserOrGroup:
+        Number(searchParams.get("assignedUserOrGroup") || "0") || 0,
+      priority: Number(searchParams.get("priority")) || 0,
+      sortBy: searchParams.get("sortBy") || "dueDate",
+      sortOrder: (searchParams.get("sortOrder") as "asc" | "desc") || "asc",
+      status: statusParam
+        ? statusParam
+            .split(",")
+            .map((status) => parseInt(status))
+            .filter((status) => !isNaN(status))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)